packages/@aws-cdk/aws-lambda-destinations/test/destinations.integ.snapshot/aws-cdk-lambda-destinations.template.json

```diff
@@ -391,5 +391,23 @@
     "MaximumRetryAttempts": 0
    }
   }
+ },
+ "Outputs": {
+  "ExportsOutputRefSnsSqsC4810B27404A5AFF": {
+   "Value": {
+    "Ref": "SnsSqsC4810B27"
+   },
+   "Export": {
+    "Name": "aws-cdk-lambda-destinations:ExportsOutputRefSnsSqsC4810B27404A5AFF"
+   }
```

Comment:
This is going to prevent replacements. Feels a little scary, is that definitely what we want?

Reply:
I _think_ so? I think in a lot of cases we would want to prevent replacements as that could be a breaking change. If it was really necessary then you could run without the update workflow. I'm not sure what the alternative would be that wouldn't have it's own tradeoffs.